docs/guides/administrator/adding-and-managing-tenants.md

```diff
@@ -0,0 +1,144 @@
+token.
+Let's do that.
+
+### 3) Add the Public Key to a Running Opstrace Instance
+
+You can think of this step as adding a new trust anchor to the trust store of a
+running Opstrace instance.
+Just like with X.509 certificates, trusting a public
+key means that authentication proof signed with the correspond _private_ key is
+accepted (reminder: in
+[public-key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography),
+verification of authentication proof only ever needs the non-sensitive public
+key material and not the private key data—that's the beauty).
+
+So, to make an existing Opstrace instance _trust_ the authentication token
+generated in the previous step, we have to put the public key into the instance:
+
+```bash
+./opstrace ta-pubkeys-add aws showdown custom-keypair.pem
+```
+
+## Add a New Tenant in the UI
+
+Visit `https://showdown.opstrace.io/cluster/tenants` and press the `Add Tenant` button.
+Type the name of the new tenant (here: `newtenant`).
+
+![add tenant gif](../../assets/tenants-guide-add.gif)
+
+**Warning**: we do not yet do strict tenant name validation.
+To make sure things
+work, please keep the name lower case `[a-z]` for now.
+
+The Opstrace controller running in the Opstrace instance will start a number of new components and initiate a DNS reconfiguration.
+
+Effectively, we're now waiting for the DNS name
+`cortex.newtenant.showdown.opstrace.io` to become available.
+
+We can probe that from our point of view with `curl`:
+
+```bash
+curl https://cortex.newtenant.showdown.opstrace.io/api/v1/labels
+```
+
+It should take about 5 minutes for DNS name resolution errors to disappear.
+Next
+up, expect an HTTP response with status code `401`, showing the error message
+`Authorization header missing` in the response body.
+
+## Test Tenant API Authentication
+
+Let's add said header and make an example API call against the Cortex API for
+the new tenant:
+
+```bash
+$ curl -vH "Authorization: Bearer $(cat token-showdown-newtenant.jwt)" \
+    https://cortex.newtenant.showdown.opstrace.io/api/v1/labels
+...
+< HTTP/2 200
+...
+{"status":"success","data":[]}
+```
+
+Getting a `200` response (and not a `401` response) means: the authentication token provided in the request was accepted.
+The so-called _authenticator_ in the Opstrace instance extracted the public key ID from the token's header section, found a corresponding public key in its trust store (think: "set of public keys that I am configured to trust"), and then performed a cryptographic verification using that public key.
+It also confirmed that the tenant name encoded in the token matches the tenant associated with the API endpoint.
+
+You could now go ahead and take this authentication token and configure serious API clients with it, such as a Prometheus instance to be able to `remote_write` to `https://cortex.newtenant.showdown.opstrace.io/api/v1/push`.
+
+## Send Data to a new Tenant with an Integration
```

Comment:
Side note, not too relevant for this review. Language-wise / expression-wise we have to improve on this: "to a tenant" is not really good in systems thinking. Maybe "to the tenant ingest API". The tenant is not just "in the cluster/instance", but it is more than that: the _sending_ side of things belongs to it, too.

"with an Integration" is a difficult language construct, doesn't sound right; too bloaty/abstract.

Reply:
Good points.  Let's dig into how we want to express these concepts and update the docs in a separate PR.  Let's also consider defining them in the key concepts or a new glossary page.

> "with an Integration" is a difficult language construct, doesn't sound right; too bloaty/abstract.

I don't follow.  An "integration" is a feature of Opstrace; perhaps we should define it in the key concepts?  Perhaps "using an integration"?